.env.development

```diff
@@ -15,6 +15,7 @@ REACT_APP_GUN_PUBLIC_URL="https://goodgun-dev.herokuapp.com/gun"
 REACT_APP_IPFS_GATEWAYS="https://gateway.pinata.cloud/ipfs/{cid},https://{cid}.cf-ipfs.com,https://ipfs.io/ipfs/{cid},https://{cid}.ipfs.dweb.link"
 REACT_APP_IPFS_LAZY_UPLOAD=true
 REACT_APP_LOG_LEVEL=debug
+REACT_APP_CLAIM_NOTIFICATION_TEST=true
```

Comment:
i've asked option to set **interval** 
NOT true/false !

if interval set - trigger notifications in this interval
otherwise - as usual